src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -919,4 +933,59 @@ private Predicate<String> getModulePredicate() {
   public RandomNumberGenerator getRandomizer() {
     return this.populationRandom;
   }
+
+  public void reduceNumberOfHospitals() {
+    // +1 Since this includes the header
+    int hospitalNumber = Integer.parseInt(Config.get("verily.limit_hospital_number")) + 1;
+
+    CSVFormat csvFileFormat = CSVFormat.EXCEL;
+
+    // Verily CSV
+    String verilyFile = "src/main/resources/providers/hospitals_verily.csv";
+    File csvFile = new File(verilyFile);
+    if (!(csvFile.exists() && !csvFile.isDirectory())) {
+      System.out.println("ERROR: File " + verilyFile + "Does not exists.\n");
+      return;
+    }
+
+    // Verily Reduced CSV
+    String hospitalFile = "src/main/resources/" + Config.get("generate.providers.hospitals.default_file");
+    String verilyReducedFile = new File(hospitalFile).getAbsolutePath();
+    File reducedCSVFile = new File(verilyReducedFile);
+    if (!(reducedCSVFile.exists() && !reducedCSVFile.isDirectory())) {
+      System.out.println("ERROR: File " + verilyReducedFile + " Does not exists.\n");
```

Comment:
Why should this print an error if this is expected?

Reply:
I'll change it to `Warning:` what do you think?